### PR TITLE
[KOGITO-8298] Improve the Kaniko build time using a pre-warmed cache …

### PR DESCRIPTION
**Pull request by davidesalerno**
…and desired resources

See:  [KOGITO-8298](https://issues.redhat.com/browse/KOGITO-8298)

Signed-off-by: Davide Salerno <dsalerno@redhat.com>

**Description of the change:**

**Motivation for the change:**

**Checklist**

- [x] Add or Modify a unit test for your change
- [x] Have you verified that tall the tests are passing?

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -105,3 +104,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/kiegroup/container-builder v0.0.0-20221028160425-7921fe1891f1 => github.com/davidesalerno/kogito-builder v0.0.0-20221209101055-e9ac23a06d2d
```
Comment (radtriste) [inline go.mod:108]: do not forget to remove before merging

**Review by radtriste — APPROVED** [review]

**Review by desmax74 — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (davidesalerno) [reply go.mod]: Updated after merging the other PR on the container-builder side!

**Review by spolti — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]
Should we take a look at the CI?

**Comment by davidesalerno** [comment]
> Should we take a look at the CI?

Probably the rebase on main branch caused some issue, I'm looking at that!

**Comment by radtriste** [comment]
https://github.com/kiegroup/kogito-serverless-operator/pull/50/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6
This is the update done this morning on main

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**